net/openr/Makefile

```diff
@@ -0,0 +1,54 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openr
+PKG_SOURCE_DATE:=2019-06-14
+PKG_SOURCE_VERSION:=79dab3d3c1bd612052d7b61e1aa01c289ed177d9
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_SOURCE_DATE).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/facebook/openr/tar.gz/$(PKG_SOURCE_VERSION)?
+PKG_HASH:=133a15d0cd7fe48bce9a922c9fe2854bfe3104c6cbde94db595a3c6d108b923f
+
+PKG_MAINTAINER:=Amol Bhave <ambhave@fb.com>
+
+PKG_BUILD_DEPENDS:=fbthrift/host
+PKG_BUILD_DIR:=$(BUILD_DIR)/openr-$(PKG_SOURCE_VERSION)
+PKG_BUILD_PARALLEL:=1
+CMAKE_BINARY_SUBDIR:=build
+CMAKE_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/openr
+	SECTION:=network
+	CATEGORY:=Network
+	TITLE:=OpenR: Distributed platform for building autonomic network functions.
+	DEPENDS:=+libatomic +libzmq-nc +libnl-core +libnl-route +re2 +fbzmq
```

Comment:
So fbzmq contains libzmq. This is adding -nc. These two packages conflict, no?

Reply:
libzmq is a shim, both libzmq-nc and libzmq-curve "provide" libzmq. By default libzmq-nc gets selected when you select libzmq, so you're right I don't need this dep here.